src/ICeWall/doc/README.md

```diff
@@ -0,0 +1,97 @@
+# ICeWall
+
+## Introduction
+
+ICeWall is a utility to place IO cells around the periphery of a design, and associate the IO cells with those present in the netlist of the design.
+
+Additional data is required to define the IO cells in the library over and above that specified in the LEF description. e.g.
+* Define the orientation of IO cells on each side
+* Allow different cell types to be used for the same type of signal depending upon which side of the die the IO cell is to be placed on. Technologies that require the POLY layer to be aligned in the same direction across the whole die require different cell variants for left/right edges and top/bottom edges
+* Define the pins that connect through the sides of the pad cells by abutment
+* Define cells that are used as breaker cells around the padring, defining the signals that they break.
+* Define the filler cells and corner cells to be used.
+
+In order to allow for a given padring to be reused across multiple designs, the definition of the placement of padcells is done independently of the signals associated with the IO cells. It is expected that the incoming design will instantiate IO cells for all IO cells that include discrete devices i.e. signal IO as well as power/ground pads. 
+
+The package data file defines where padcells are to be placed, but the association between the placement and the netlist not made until the signal mapping file is read in. This separation of placement and netlist information allows the same IO placement to be re-used for multiple designs. A package data file can also be extracted from an existing DEF file to allow padrings of existing designs to be reused.
+
+The signal mapping file associates signals in the netlist with locations in the padring
+
+## Data definitions
+
+ICeWall acts upon the information provided in the data files to produce a padring layout. The required data is split across 3 files to allow for the anticipated use models
```

Comment:
Maybe update to clarify that only the library data file is needed when using the TCL interface commands.